tests/tools/traffic-replay/RandomReplay.py

```diff
@@ -0,0 +1,167 @@
+#!/bin/env python3
+'''
+'''
+#  Licensed to the Apache Software Foundation (ASF) under one
+#  or more contributor license agreements.  See the NOTICE file
+#  distributed with this work for additional information
+#  regarding copyright ownership.  The ASF licenses this file
+#  to you under the Apache License, Version 2.0 (the
+#  "License"); you may not use this file except in compliance
+#  with the License.  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import gevent
+import socket
+import requests
+import os
+from threading import Thread
+import sys
+from multiprocessing import current_process
+import sessionvalidation.sessionvalidation as sv
+import lib.result as result
+import extractHeader
+import mainProcess
+import json
+import gzip
+bSTOP = False
+def createDummyBodywithLength(numberOfbytes):
+    if numberOfbytes<=0:
+        return None
+    body= 'a'
+    while numberOfbytes!=1:
```

Comment:
Is there no faster way to do this? It seems like if the size is non-trivial this will be very expensive.